R/drive_delete.R

```diff
@@ -32,6 +33,5 @@ delete_one <- function(id) {
     endpoint = "drive.files.delete",
     params = list(fileId = id)
   )
-  response <- make_request(request)
-  process_response(response)
+  do_request(request)
```

Comment:
I propose that we NOT indulge in `do_request()` but rather keep doing `make_request()` then `process_response()`. On principle, I think future debugging will be easier if we keep individual steps.